src/chrome/content/rules/TomsHardware.com.xml

```diff
@@ -0,0 +1,18 @@
+<!--
+	Certificate mismatch: 
+		- tomshardware.com
+	
+	Expired certificate:
+		- gb.tomshardware.com
+		- dld.tomshardware.com
+-->
+<ruleset name="TomsHardware.com">
+	<target host="tomshardware.com" />
+	<target host="www.tomshardware.com" />
+	
+	<securecookie host=".+" name=".+" />
+	
+	<rule from="^http://tomshardware\.com/" to="https://www.tomshardware.com/" />
+	
+	<rule from="^http:"to="https:" />
```

Comment:
You're missing a space before `to` here.

Reply:
Thanks for pointing it out. Fixed it now.